example/metric/exponential_histogram/example.go

```diff
@@ -0,0 +1,322 @@
+		mexporter.WithMetricDescriptorTypeFormatter(workloadMetricsPrefixFormatter),
+	)
+	if err != nil {
+		log.Fatalf("Failed to create exporter: %v", err)
+	}
+
+	clabels := []attribute.KeyValue{attribute.Key("key").String("value")}
+	defaultBuckets := []float64{0, 5, 10, 25, 50, 75, 100, 250, 500, 1000}
+	linearBuckets := make([]float64, 35)
+	for i := 0; i < 35; i++ {
+		linearBuckets[i] = float64(i*10 + 10)
+	}
+
+	viewLatencyA := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency",
+		},
+		sdkmetric.Stream{
+			Name: "latency_a",
+			Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{
+				MaxSize:  160,
+				MaxScale: 20,
+			},
+		},
+	)
+
+	viewLatencyB := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency",
+		},
+		sdkmetric.Stream{
+			Name: "latency_b",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: linearBuckets,
+			},
+		},
+	)
+
+	viewLatencyC := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency",
+		},
+		sdkmetric.Stream{
+			Name: "latency_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	viewLatencyShiftedA := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_a",
+			Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{
+				MaxSize:  160,
+				MaxScale: 20,
+			},
+		},
+	)
+
+	viewLatencyShiftedB := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_b",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: linearBuckets,
+			},
+		},
+	)
+
+	viewLatencyShiftedC := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	viewLatencyMultimodalA := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_a",
+			Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{
+				MaxSize:  160,
+				MaxScale: 20,
+			},
+		},
+	)
+
+	viewLatencyMultimodalB := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_b",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: linearBuckets,
+			},
+		},
+	)
+
+	viewLatencyMultimodalC := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	// initialize a MeterProvider with that periodically exports to the GCP exporter.
+	provider := sdkmetric.NewMeterProvider(
+		sdkmetric.WithView(viewLatencyA),
```

Comment:
nit: WithView can take more than 1 view.  You can pass all views in a single option

Reply:
Fixed